website/docs/r/aviatrix_spoke_gateway.html.markdown

```diff
@@ -99,25 +99,24 @@ The following arguments are supported:
 
 ~> **NOTE:** Custom SNAT and DNAT support have been deprecated and functionality has been moved to **aviatrix_gateway_snat** and **aviatrix_gateway_dnat** respectively, in provider version R2.10. Please see notes for `snat_mode`, `snat_policy` and `dnat_policy` in the Notes section below.
 
+### Encryption
+* `enable_encrypt_volume` - (Optional) Enable EBS volume encryption for Gateway. Only supports AWS. Valid values: true, false. Default value: false.
```

Comment:
is this released? Can we rename this parameter to 'enable_volume_encryption'?

Reply:
This has been released back in R2.9